pkg/web/metricsHandler.go

```diff
@@ -23,11 +24,13 @@ func (mh *metricsHandler) CreateMetric(w http.ResponseWriter, r *http.Request) {
 
 	// decode the client payload into the metric var
 	if err := json.NewDecoder(r.Body).Decode(&metric); err != nil {
-		boom.BadRequest(w, "Invalid Data")
+		log.WithFields(log.Fields{"error": err.Error()}).Error("error parsing client payload")
+		boom.BadRequest(w, "invalid JSON payload")
 		return
 	}
 
 	if valid, reason := metric.Validate(); !valid {
+		log.WithFields(log.Fields{"reason": reason}).Info("invalid client payload")
```

Comment:
Seems we're on the same page here. 
Info is good here as its a client error, and the error is sent back to the client to do something about it.
Also, it wouldn't be polluting the server unnecessarily when there's nothing the server admin can do.